src/pages/patterns/login-pattern/index.mdx

```diff
@@ -0,0 +1,181 @@
+1. **Forgot IBMid?:** Help link specifically for IBMid questions and issues.
+1. **Remember me option**: Saves the IBMid and presents the completed input field the next time the user logs in.
+1. **Continue button**: The primary call to action. When clicked, the email address is validated and routes the user to either the federated or non-federated flow.
+1. **Create account option:** Link to URX form to create an account.
+1. **Need help?:** Help link specifically for IBMid questions and issues.
+
+### When to use
+
+The login page is presented to users in the following scenarios:
+
+- When a user wants to gain access to an app.
+- When a user has logged out voluntarily. They will see a confirmation message after which they will be automatically redirected back to the login page.
+- When a user has been logged out due to inactivity. In this scenario, when the user logs back in they should be redirected to the last page they were on, before being logged out.
+
+## Behavior
+
+Federated users log in with SSO (single sign-on) and non-federated users are required to provide both their IBMid and password. Separate the IBMid and password fields into different flows to easily accommodate both federated and non-federated users.
+
+IBM uses multi-factor login for products. The IBMid should be requested upfront with a “Continue” button to move forward. This allows the system to distinguish which path the user needs to take in the background instead of making the user read through options and choose. The system validates the username and provides a field for the password only if required.
+
+<Row>
+<Column colLg={8}>
+
+![IBMid input field](/images/IBMid_field.png)
+
+<Caption>
+  Step 1 of the login flow for both federated and non-federated users
+</Caption>
+
+</Column>
+</Row>
+
+From this point, the user will either continue to their company’s SSO login experience or they’ll see a password field and a “Log in” button. The password page should include a way to return to the prior page in case they make a mistake while filling out their IBMid and a “Forgot password” button. When users enter their authentication information progressively, the cognitive load is reduced.
+
+<Row>
+<Column colLg={8}>
+
+![IBMid input field](/images/Login_password_field.png)
+
+<Caption>Step 2 of the login flow for non-federated users</Caption>
+
+</Column>
+</Row>
+
+## Design types
+
+The Login page should focus on the main task of accessing one’s account. The point of this page is to get the user into their account quickly without complicating the workflow. It’s not recommended to include anything that will create too much of a distraction from the form.
+
+Simple illustrations can add personality and help differentiate the form from other products, but they should not pull the user's focus away from logging in. Consider the simplest approach to logging into an account on mobile and maintain that simplicity on tablet and desktop.
```

Comment:
Same questions about "logging in to" vs. "loggin into"

```suggestion
Simple illustrations can add personality and help differentiate the form from other products, but they should not pull the user's focus away from logging in. Consider the simplest approach to logging in to an account on mobile and maintain that simplicity on tablet and desktop.
```

Reply:
Looks like "log in to" is generally thought to be the better move: https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to